tools/integration_tests/readonly/delete_objects.sh

```diff
@@ -0,0 +1,3 @@
+gsutil rm gs://$TEST_BUCKET/Test/a.txt
```

Comment:
why not just delete everything from bucket

Reply:
I though we should remove the objects only which we have created.